src/pages/top.js

```diff
@@ -197,21 +197,21 @@ function Top({ trees, planters, countries, organizations }) {
   );
 }
 
-export async function getServerSideProps() {
+export async function getStaticProps() {
```

Comment:
@okdv can you explain this a little more? What would be the behavior using this way?

Reply:
@dadiorchen sure thing

getServerSideProps calls that data at load time from the client side (to my understanding, what we are trying to avoid for this page)

getStaticProps generates a static, server side rendered page at build time. This static data will be re-rendered based on the rule provided in the revalidation rule

there is a lot more in depth info in the nextjs docs though, so i'll defer to that https://nextjs.org/docs/basic-features/data-fetching/overview

Reply:
@okdv perfect